bin/copy-labels.js

```diff
@@ -0,0 +1,282 @@
+        `from repository '${args.srcRepo}' to file '${args.dstFile}'`
+    );
+    printLabels(labels);
+  } else {
+    const existingLabels = await handler.get();
+    let msg = 'Copy the following labels from';
+    if (args.srcFile) {
+      msg += ` file '${args.srcFile}'`;
+    } else {
+      msg += ` repository '${args.srcRepo}'`;
+    }
+    msg += ` to repository '${args.dstRepo}'`;
+    if (args.update) msg += ' updating existing labels';
+    console.log(msg);
+    printAddedLabels(args, labels, existingLabels, usedLabels);
+    if (args.update) {
+      printUpdatedLabels(args, labels, existingLabels, usedLabels);
+    }
+    printUnchangedLabels(args, labels, existingLabels, usedLabels);
+  }
+};
+
+const copyLabels = async (args, handler, labels) => {
+  if (args.dstFile) {
+    await writeDump(args.dstFile, labels);
+  } else {
+    try {
+      labels = await handler.copyFrom(labels, args.update);
+    } catch (err) {
+      helpExit('Cannot copy labels', err);
+    }
+  }
+  return labels;
+};
+
+const getLabels = async (args, handler) => {
+  let labels;
+  if (args.srcFile) {
+    labels = await readDump(args.srcFile);
+  } else {
+    try {
+      labels = await handler.getFrom(args.srcRepo);
+    } catch (err) {
+      helpExit('Cannot read dump', err);
+    }
+  }
+  return labels;
+};
+
+const getConfig = async args => {
+  if (args.config) {
+    const { user, token } = await readDump(args.config);
+    args = { user, token, ...args };
+  }
+
+  const [src, dst] = args.args;
+  if (!src) helpExit('missing source repository or file');
+  if (!dst) helpExit('missing destination repository or file');
+
+  if (path.extname(src) === '.json') args.srcFile = src;
+  else args.srcRepo = src;
+  if (path.extname(dst) === '.json') args.dstFile = dst;
+  else args.dstRepo = dst;
+
+  if (args.srcFile && args.dstFile) {
+    helpExit('Either source or destination should be repository');
+  }
+  return args;
+};
+
+const run = async args => {
+  args = await getConfig(args);
+  const handler = new LabelHandler({
+    repo: args.dstRepo,
+    user: args.user,
+    token: args.token,
+  });
+
+  if (args.dry) console.log('This is a dry run, no changes will be made\n');
+
+  let usedLabels = new Map();
+  if (args.dstRepo && args.delete) {
+    try {
+      if (args.dry) usedLabels = await dryDeleteLabels(args, handler);
+      else usedLabels = await handler.delete();
+    } catch (err) {
+      helpExit(`Cannot delete labels${args.dry ? '(dry)' : ''}`, err);
+    }
+  }
+  if (usedLabels.size !== 0) {
+    console.error(
+      `The following labels have related issues and won't be deleted`
+    );
+    console.table(usedLabels);
+  }
+
+  const labels = await getLabels(args, handler);
+  if (args.dry) {
+    await dryCopyLabels(args, handler, labels, usedLabels);
+  } else {
+    await copyLabels(args, handler, labels);
+    printLabels(labels);
+  }
+  return labels;
+};
+
+const args = handleArgs();
+run(args).then(
+  () => {},
```

Comment:
What's the point of passing an empty function here?
```suggestion
  null,
```

Reply:
Also, you can just use `Promise.prototype.catch()` here.